src/firefox/index.js

```diff
@@ -44,26 +47,26 @@ export function run(
       return new Promise((resolve) => {
         let firefox = results.process;
 
-        console.log(`Executing Firefox binary: ${results.binary}`);
-        console.log(`Executing Firefox with args: ${results.args.join(' ')}`);
+        log.debug(`Executing Firefox binary: ${results.binary}`);
+        log.debug(`Executing Firefox with args: ${results.args.join(' ')}`);
 
         firefox.on('error', (error) => {
           // TODO: show a nice error when it can't find Firefox.
           // if (/No such file/.test(err) || err.code === 'ENOENT') {
-          console.log(`Firefox error: ${error}`);
+          log.error(`Firefox error: ${error}`);
           throw error;
         });
 
         firefox.stderr.on('data', (data) => {
-          console.error(`stderr: ${data.toString().trim()}`);
+          log.error(`stderr: ${data.toString().trim()}`);
         });
 
         firefox.stdout.on('data', (data) => {
-          console.log(`stdout: ${data.toString().trim()}`);
+          log.info(`stdout: ${data.toString().trim()}`);
         });
 
         firefox.on('close', () => {
-          console.log('Firefox closed');
+          log.debug('Firefox closed');
```

Comment:
for normal operation, Firefox will close because the user said so. Thus, I don't think it needs to be an info message. The error callback is already logged (in the case of a crash).